src/EditorFeatures/VisualBasicTest/DocumentationComments/DocumentationCommentTests.vb

```diff
@@ -1,9 +1,11 @@
 ﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
 Imports Microsoft.CodeAnalysis.Editor.Host
+Imports Microsoft.CodeAnalysis.Editor.UnitTests
 Imports Microsoft.CodeAnalysis.Editor.UnitTests.DocumentationComments
 Imports Microsoft.CodeAnalysis.Editor.UnitTests.Workspaces
 Imports Microsoft.CodeAnalysis.Editor.VisualBasic.DocumentationComments
+Imports Microsoft.CodeAnalysis.Editor.VisualBasic.LineCommit
```

Comment:
Do we need these line to be added? If not, let us revert.

Reply:
They are needed now.